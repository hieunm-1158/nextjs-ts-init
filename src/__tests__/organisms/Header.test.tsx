import { render } from '@testing-library/react';

import Header from '@/organisms/Header';

describe('organisms/Header', () => {
  describe('snapshot', () => {
    it('should render and match the snapshot', () => {
      const { container } = render(<Header />);
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
