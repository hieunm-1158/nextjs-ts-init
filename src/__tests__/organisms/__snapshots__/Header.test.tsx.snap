// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`organisms/Header snapshot should render and match the snapshot 1`] = `
<nav
  class="border-gray-200 bg-white dark:bg-gray-900"
>
  <div
    class="mx-auto flex max-w-screen-xl flex-wrap items-center justify-between p-4"
  >
    <a
      class="flex items-center"
      href="/"
    >
      <img
        alt="Flowbite Logo"
        class="mr-3"
        data-nimg="1"
        decoding="async"
        height="32"
        loading="lazy"
        src="https://flowbite.com/docs/images/logo.svg"
        style="color: transparent;"
        width="32"
      />
      <span
        class="self-center whitespace-nowrap text-2xl font-semibold dark:text-white"
      >
        Flowbite
      </span>
    </a>
    <button
      aria-controls="navbar-default"
      aria-expanded="false"
      class="inline-flex h-10 w-10 items-center justify-center rounded-lg p-2 text-sm text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-gray-200 dark:text-gray-400 dark:hover:bg-gray-700 dark:focus:ring-gray-600 md:hidden"
      data-collapse-toggle="navbar-default"
      type="button"
    >
      <span
        class="sr-only"
      >
        Open main menu
      </span>
      <svg
        aria-hidden="true"
        class="h-5 w-5"
        fill="none"
        viewBox="0 0 17 14"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M1 1h15M1 7h15M1 13h15"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
        />
      </svg>
    </button>
    <div
      class="hidden w-full md:block md:w-auto"
      id="navbar-default"
    >
      <ul
        class="mt-4 flex flex-col rounded-lg border border-gray-100 bg-gray-50 p-4 font-medium dark:border-gray-700 dark:bg-gray-800 md:mt-0 md:flex-row md:space-x-8 md:border-0 md:bg-white md:p-0 md:dark:bg-gray-900"
      >
        <li>
          <a
            aria-current="page"
            class="block rounded bg-blue-700 py-2 pl-3 pr-4 text-white dark:text-white md:bg-transparent md:p-0 md:text-blue-700 md:dark:text-blue-500"
            href="/"
          >
            home.title
          </a>
        </li>
        <li>
          <a
            class="block rounded py-2 pl-3 pr-4 text-gray-900 hover:bg-gray-100 dark:text-white dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-blue-500"
            href="/service"
          >
            service.title
          </a>
        </li>
        <li>
          <a
            class="block rounded py-2 pl-3 pr-4 text-gray-900 hover:bg-gray-100 dark:text-white dark:hover:bg-gray-700 dark:hover:text-white md:border-0 md:p-0 md:hover:bg-transparent md:hover:text-blue-700 md:dark:hover:bg-transparent md:dark:hover:text-blue-500"
            href="/contact"
          >
            contact.title
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
